### DISCUSSION TITLE
家里上github有问题

### DISCUSSION THREAD

**Discussion by eightsheep67** [vernesong/OpenClash#2880] [Q&A] [answered]
最近才发现家里上github有问题。家里用openclash，还建立了简单的socks供我手机上小飞机使用。

具体现象就是：

1.无法打开github，nslookup github.com 是182.43.124.6，指向了国家反诈；
nslookup www.github.com 是192.30.255.112，美国地址，直接访问该ip显示不安全，继续的话无法访问

2.关掉openclash，刷新dns后可以访问github.com，指向20.205.243.166，新加坡地址，直接访问该ip显示不安全，继续的话进入github主页。

3.手机不连wifi通过联通数据，打开小飞机访问github.com正常

4.我想会不会是我openclash中设置的哪个dns出问题了,就用用nslookup域名后带dns地址方法来查询，但是无论用运营商自己的dns，还是其他所有的公共dns比如114dns，8.8.8.8，223.5.5.5，在开openclash的时候，nslookup查出来的都是和1一样的地址；关掉openclash查出来的都是和2一样的地址。貌似和dns也没有关系。

这到底会是啥问题？？

**Comment by eightsheep67** [comment]
切换成FAKE IP就可以了，切换会redir-host兼容又不行了？？？所以redir-host是有什么问题吗？

**Comment by kingsey** [comment, marked as answer]
最近有大面积dns污染，把github或者182.43.124.6加到fallback-filter里吧
